docker/Makefile

```diff
@@ -0,0 +1,36 @@
+SHELL = bash
```

Comment:
Maybe as a follow up PR (and issue), can we get rid of Make and use bazel docker rules to start/stop nodes?

Reply:
https://github.com/liftedinit/many-rs/issues/312